### DISCUSSION TITLE
Support for Netconf 1.1?

### DISCUSSION THREAD

**Discussion by kgovind12** [sshnet/SSH.NET#1171] [Q&A]
Does this library support Netconf 1.1? If so, which version of the library supports it? So far I have seen only Netconf 1.0.

**Comment by WojciechNagorski** [comment]
We support 1.1 https://github.com/sshnet/SSH.NET/blob/cd1151decac9f5672411eebb12d939cedc53b317/src/Renci.SshNet/Netconf/NetConfSession.cs#L138
If you have a problem please report it.
